src/App.js

```diff
@@ -1,10 +1,79 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
+import Button from './components/Button';
+import Pokecard from './components/Pokecard';
+import pokemonSVG from './assets/International_Pokémon_logo.svg';
+import axios from 'axios';
 
 function App() {
+  // twee states 1 voor het pokemon object, de ander voor het overkoepelende data object
+  const [pokemons, setPokemons] = useState([]);
+  const [data, setData] = useState({});
+
+  // functie voor het verkrigjen van de 20 pokemon, en het overkoepelende data object om gebruik te maken van next en previous
+  async function fetchData(link) {
+    try {
+      const result = await axios.get(link);
+      setPokemons(result.data.results);
+      setData(result);
```

Comment:
Uiteindelijk staat de lijst met pokemons opgeslagen in twee variabelen. Je zou daarom er ééntje weg kunnen doen